doc/rst/tools/mason/guide/manifestfile.rst

```diff
@@ -16,33 +16,136 @@ For example, ``Mason.toml``:
     chplVersion = "1.16.0"
     license = "None"
     name = "MyPackage"
+    type = "application"
     version = "0.1.0"
 
     [dependencies]
     curl = '1.0.0'
 
-The ``chplVersion`` field indicates Chapel releases compatible with this
-package. There are a number of accepted formats:
 
-.. code-block:: text
-
-    "1.16.0"         # 1.16.0 or later
-    "1.16"           # 1.16.0 or later
-    "1.16.0..1.19.0" # 1.16 through 1.19, inclusive
-
-By default, ``chplVersion`` is set to represent the current Chapel release or
-later. For example, if you are using the 1.16 release, chplVersion will be
-``1.16.0``.
-
-The ``license`` field indicates the software license under which the package is distributed.
-Any of the licenses available at the `SPDX License List <https://spdx.org/licenses/>`_ can be used for Mason packages.
-The license field defaults to ``None``.
+Format
+~~~~~~
 
 TOML is the configuration language chosen by the chapel team for
 configuring programs written in chapel. A TOML file contains the
 necessary information to build a chapel program using mason.
 View documentation for the TOML format here: `TOML Spec <https://github.com/toml-lang/toml>`_.
 
+Mason understands specific fields in the TOML file, which are described below.
+
+* ``[brick]``: This is the root table of the manifest file. It contains metadata
+  about the package itself.
+
+   * ``name``: The name of the package. This should be a unique identifier for
+     the package.
+
+   * ``type``: The type of the package. This can be
+     :ref:`application <mason-applications>`, :ref:`library <mason-libraries>`, or
+     :ref:`light <mason-lightweight-projects>`.
+
+   * ``version``: The version of the package, following semantic versioning
+     (e.g., "0.1.0").
+
+   * ``authors``: A list of authors of the package, formatted as strings.
+     Each author can be represented as a string with their name and
+     email address.
+
+   * ``chplVersion``: Indicates Chapel releases compatible with this
+     package. There are a number of accepted formats:
+
+        .. code-block:: text
+
+           "1.16.0"         # 1.16.0 or later
+           "1.16"           # 1.16.0 or later
+           "1.16.0..1.19.0" # 1.16 through 1.19, inclusive
+
+     By default, ``chplVersion`` is set to represent the current Chapel release or
+     later. For example, if you are using the 1.16 release, chplVersion will be
+     ``1.16.0``.
```

Comment:
While we're here, it might be nice to replace "1.16" with "2.0" instead

Reply:
I switched it to 2.1 for a slightly more interesting number, `2.0.0` blends together